projects/m1/011-note-to-frequency/python/main.py

```diff
@@ -0,0 +1,25 @@
+note = input('type a note: ').lower()
+octave = int(input('type an octave: '))
+
+if note == 'c':
+    frequency = 261.63 / 2**(4-octave)
+    print(frequency)
```

Comment:
It might be a good idea to save the frequency in a variable and print it at the end _(so you call the print function once)_